### DISCUSSION TITLE
Show a zero in cells that have zero value

### DISCUSSION THREAD

**Discussion by itm-platform** [exceljs/exceljs#2519] [Ideas & Proposals]
Hi there. I need to hide the zeroes, which is a worksheet property accessible via Options in Excel
Can I do this with exceljs?
![image](https://user-images.githubusercontent.com/48089993/65889333-67aad700-e3a1-11e9-9273-6e0bd9b1021a.png)

**Comment by sunhao12** [comment]
i also have this problem   
when i unchecked this option  
then export excel by exceljs    
this option is auto checked
